build_ovtf.py

```diff
@@ -192,11 +174,6 @@ def main():
     # Default directories
     build_dir = arguments.build_dir
 
-    if (arguments.use_tensorflow_from_location != '' and
```

Comment:
Is using TF from location removed?

Reply:
build_tf_from_source option through build_ovtf.py script is removed, now when required tf needs to be built separately using build_tf.py